test/unit/krane/kubernetes_resource/custom_resource_definition_test.rb

```diff
@@ -96,6 +96,7 @@ def test_cr_instance_fails_validation_when_rollout_conditions_for_crd_invalid
       logger: @logger, statsd_tags: @statsd_tags, crd: crd,
       definition: {
         "kind" => "UnitTest",
+        "apiVersion" => "stable.example.io/v1",
```

Comment:
`apiVersion` is required of any k8s resource, so it makes more sense to add it here than try to code around its absence